### ISSUE TITLE
Night mode inverts also picture colors on Android 4.2

### ISSUE THREAD

**Issue by kelson42** [kiwix/kiwix-android#153]
Tested on BitBar with a 

```
    Samsung Galaxy S III Mini GT-I8200L
    ANDROID
    Dual-core 1.2 GHz / ARMv7
    1024 MB
    WVGA (800 x 480)
```

**Comment by mhutti1** [comment]
Do you mean 4.2?

**Comment by kelson42** [comment]
@mhutti1 yes

**Comment by dbedrenko** [comment]
Any status update on this? Otherwise, could I get a hint of where in the code I should even look to find the problem? Should I be fixing this in the Android port or in the "core" libs?

I just got a 128GB microSD JUST so that I could have wikipedia with pictures, and it's useless when they are inverted.

**Comment by kelson42** [comment]
@dbedrenko Which version of Android do you have?

**Comment by dbedrenko** [comment]
Full specs:
```
Android v4.2.2
Model: Galaxy Tab 2, GT-P5100
Kernel: v3.0.31

Kiwix: v2.2 Build 53
```

I bought this tablet exclusively for reading, so you can bet your chops I'm motivated to find a fix for this :)

**Comment by mhutti1** [comment]
It seems the problem is that our current solution https://github.com/kiwix/kiwix-android/blob/master/app/src/main/java/org/kiwix/kiwixmobile/ZimContentProvider.java#L457 does not work on earlier android versions I assume because of the webkit version. We need to come up with an alternative to invert the image colours that does.

**Comment by kelson42** [comment]
I suspect this bug is not really easy to fix and as Android 4.2 does only represent 5% of the overall Android audience I propose to close it as WONTFIX.
